tools/preflight-check.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+import sys
+import csv
+
+SUPPORTED_STORES = ["amazon", "battlenet", "none", "egs", "ubisoft", "ea", "humble", "itchio", "steam", "gog", "zoomplatform"]
+
+def main():
+    file = sys.argv[1]
+    with open(file, 'r') as csvfile:
+        rows = csv.reader(csvfile)
+        header = True
+        release_ids = list()
+        for i, row in enumerate(rows, 1):
+            if header:
+                header = False
+                continue
+
+            if not row:
+                print("Empty row found:", i)
+                exit(1)
+
+            title = row[0]
+            store = row[1]
+            codename = row[2]
+            ulwgl_id = row[3]
+
+            if not (title or store or codename or ulwgl_id):
```

Comment:
Previous comment on this line:

> If you want to pull `not` in front, replace `or` with `and` between the conditions.

Sorry for nagging, but please recheck the logic in this line. As it is now, it will only trigger if all required fields are missing and result in false.

Reply:
Ah, yes. I somehow missed it